samples/ustream_forward_basic/src/main.c

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+#include "az_ulib_result.h"
+#include "az_ulib_ustream_forward.h"
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define START_MEMORY_OFFSET 0
+static const char ustream_forward_producer_buf[] = "Hello World!\r\n";
+
+void flush_callback(
+    const uint8_t* const buffer,
+    size_t size,
+    az_ulib_callback_context flush_callback_context);
+az_result my_consumer(void);
+
+typedef struct ustream_forward_basic_context
+{
+  offset_t offset;
+  char buffer[16];
```

Comment:
>16 [](http://example.com/codeflow?start=14&length=2)

I would make it bigger, like 100, to aboid any confusion with the size of `ustream_forward_producer_buf`.